### PR TITLE
Story 21

### PR DESCRIPTION
**Pull request by amhursh**
Related Issue: #7 

Summary: A whole lot:

1) Added list of all orders to admin dashboard
2) List order totals by status
3) Add links to orders
4) Added ability to filter view of orders by status
5) Added ability to update each order's status

Author(s): amhursh

### CODE REVIEW

File: app/controllers/admin/admin_controller.rb [hunk 1]
```diff
@@ -2,7 +2,17 @@ class Admin::AdminController < Admin::BaseController
   before_action :require_admin
 
   def show
-    
+    if params[:orders] == 'ordered'
```
Comment (limsammy) [inline app/controllers/admin/admin_controller.rb:5]: We should probably break this out to a helper or at least to the model level.

File: app/controllers/orders_controller.rb [hunk 2]
```diff
@@ -20,4 +21,13 @@ def create
 			redirect_to cart_path
 		end
 	end
+
+	def update
+		order = Order.find(params[:id])
+		order.cancelled! if params[:status] == 'cancelled'
```
Comment (limsammy) [inline app/controllers/orders_controller.rb:27]: Yeah, we should've thought ahead to make this RESTful. Looks like you got it up and running though! Nice work

File: app/views/admin/admin/show.html.erb [hunk 3]
```diff
@@ -36,11 +36,34 @@
...
+    </div>
+
+    <div class="dashboard_orders">
+        <% @orders.each do |order| %>
+        <div class="admin_index_order<%= order.id %>">
+            Id: <%= order.id %>
+            Created At: <%= order.created_at %>
+            Updated At: <%= order.updated_at %>
+            Status: <%= order.status %>
+            <%= link_to "View Order", order_path(order) %>
```
Comment (limsammy) [inline app/views/admin/admin/show.html.erb:62]: This all looks pretty ugly and beefy. Let's talk about possible refactors? Can't think of any right now lol.

File: spec/features/admin/admin_can_see_all_orders_spec.rb [hunk 4]
```diff
@@ -0,0 +1,118 @@
+require 'rails_helper'
+
+describe "Admin can see and manipulate all orders" do
```
Comment (limsammy) [inline spec/features/admin/admin_can_see_all_orders_spec.rb:3]: WHAT A TEST

File: spec/features/admin/admin_cannot_modify_users_spec.rb [hunk 5]
```diff
@@ -55,8 +55,6 @@
 
           visit("/users/5")
 
-          save_and_open_page
```
Comment (limsammy) [inline spec/features/admin/admin_cannot_modify_users_spec.rb:58]: damnit randy

**Review by limsammy — APPROVED** [review]
Looks good to merge. Will wait on Randy and Joel

**Comment by JoelLindow** [comment]
LOOKS GOOD GUYS

**Pull request merged**